src/pages/index.js

```diff
@@ -1,12 +1,12 @@
-import React from "react"
+import React, { Suspense } from "react"
```

Comment:
Do you plan use Suspense in this file in the future?

Reply:
No, it happened occasionally. We can think over adding a rule to eslint.